LoopOnboardingKitUI/View Controllers/OnboardingUICoordinator.swift

```diff
@@ -242,34 +242,39 @@ class OnboardingUICoordinator: UINavigationController, CGMManagerOnboarding, Pum
     }
 
     private func constructTherapySettingsViewModel(therapySettings: TherapySettings) -> TherapySettingsViewModel? {
+        return TherapySettingsViewModel(therapySettings: therapySettings, delegate: self)
+    }
+}
+
+extension OnboardingUICoordinator: TherapySettingsViewModelDelegate {
+    func syncBasalRateSchedule(items: [RepeatingScheduleValue<Double>], completion: @escaping (Result<BasalRateSchedule, Error>) -> Void) {
+        // noop
```

Comment:
Since these two are commonly NOPs for onboarding, considering making the NOP the default in an extension in LoopKit.

Reply:
Would recommend not creating default implementations for protocols: https://betterprogramming.pub/swift-why-you-should-avoid-using-default-implementations-in-protocols-eeffddbed46d#:~:text=As%20mentioned%20in%20Swift's%20documentation,can%20also%20add%20new%20ones.